apps/nayre/package.json

```
@@ -14,15 +14,15 @@
 		"lint": "eslint --fix --ext ts src"
 	},
 	"dependencies": {
-		"@discordjs/builders": "^1.4.0",
-		"@discordjs/collection": "^1.3.0",
+		"@discordjs/builders": "^1.5.0",
+		"@discordjs/collection": "^1.4.0",
 		"@skyra/env-utilities": "^1.1.0",
 		"@skyra/http-framework": "^0.15.3",
 		"@skyra/http-framework-i18n": "^0.6.0",
```

```suggestion
		"@skyra/http-framework-i18n": "^0.6.0",
		"@skyra/internal": "workspace:^",
```

And run `yarn` in the root afterwards. Fixes build.